packages/expo-cli/src/commands/upload/submission-service/__mocks__/SubmissionService.ts

```diff
@@ -0,0 +1,43 @@
+import { v4 as uuid } from 'uuid';
+
+import {
+  Platform,
+  StartSubmissionResult,
+  Submission,
+  SubmissionConfig,
+  SubmissionStatus,
+} from '../SubmissionService.types';
+
+const SubmissionService = {
+  startSubmissionAsync,
+  getSubmissionAsync,
+};
+
+const DEFAULT_CHECK_INTERVAL_MS = 0;
+
+const submissionStore: Record<string, Submission> = {};
+
+async function startSubmissionAsync(
+  platform: Platform,
+  _config: SubmissionConfig
+): Promise<StartSubmissionResult> {
+  const id = uuid();
+  const submission: Submission = {
+    id,
+    accountId: uuid(),
+    userId: uuid(),
+    platform,
+    status: SubmissionStatus.IN_QUEUE,
+    createdAt: new Date(),
```

Comment:
Those values should be created when inserting to db, if just want to satisfy type I would suggest to extend Submission and make it optional.

Reply:
Have you noticed that it's a mocked module? :D